src/components/message/message.style.js

```diff
@@ -0,0 +1,40 @@
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import BaseTheme from '../../style/themes/base';
+import { THEMES } from '../../style/themes';
+import messageClassicStyle from './message-classic.style';
+
+const MessageStyle = styled.div`
+  position: relative;
+  display: flex;
+  justify-content: flex-start;
+  align-content: center;
+  ${({ theme, transparent, messageType }) => {
+    return theme.name !== THEMES.classic ? getBorderAndBackgroundStyles(theme, transparent, messageType) : null;
+  }}
+  ${({ theme }) => (theme.name === THEMES.classic ? messageClassicStyle : null)};
+`;
+
+function getBorderAndBackgroundStyles(theme, transparent, messageType) {
+  return `
+    border: ${transparent ? 'none;' : `1px solid ${theme.status[messageType]};`}
+    background-color: ${transparent ? `${theme.colors.white};` : `${theme.colors.white};`}
```

Comment:
This ternary returns the same thing regardless of the result of the test expression (`transparent`).